google/cloud/forseti/services/inventory/storage.py

```diff
@@ -52,6 +52,7 @@
 BASE = declarative_base()
 CURRENT_SCHEMA = 1
 PER_YIELD = 1024
+MAX_ALLOWED_PACKET = 32 * 1024 * 1024  # 32 Mb default mysql max packet size
```

Comment:
I think the default max allowed packet is 4mb in mysql 5.7
https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_max_allowed_packet

Reply:
That is the default, but this is how CloudSQL sqlproxy is configured:

```
mysql> select @@max_allowed_packet;
+----------------------+
| @@max_allowed_packet |
+----------------------+
|             33554432 |
+----------------------+
1 row in set (0.00 sec)

mysql> select @@max_allowed_packet/1024/1024;
+--------------------------------+
| @@max_allowed_packet/1024/1024 |
+--------------------------------+
|                    32.00000000 |
+--------------------------------+
1 row in set (0.00 sec)
```